client/src/modules/dashboard/Events/pages/EventsPage.tsx

```diff
@@ -248,7 +250,11 @@ export const EventsPage: NextPageWithLayout = () => {
                         {name}
                       </LinkButton>
                     </VStack>
-                    <Text>{invite_only ? 'Yes' : 'No'}</Text>
+                    {invite_only && (
+                      <Tooltip label="invite_only">
+                        <LockIcon />
+                      </Tooltip>
+                    )}
```

Comment:
```suggestion
                    {invite_only ? (
                      <HStack>
                        <Tooltip label="Invite only">
                          <LockIcon />
                        </Tooltip>
                        <Text>Yes</Text>
                      </HStack>
                    ) : (
                      <Text>No</Text>
                    )}
```
When row is present only for the invite only case, the column with values gets unaligned to the column with headers.

Reply:
I don't know why I thought I moved it to the Event Name and removed invite only rows. 🤦